hw/mcu/dialog/da1469x/include/mcu/da1469x_hal.h

```diff
@@ -94,6 +94,23 @@ const struct qspi_flash_config *da1469x_qspi_get_config(void);
  */
 void hal_os_tick_calc_params(uint32_t cycles_per_sec);
 
+/**
+ * Configure GPIO using a single value.
+ *
+ * @param pin GPIO ID
+ * @param raw_mode Combined mode / I/O properties value
```

Comment:
You could add description of what ***mode*** and ***I/O properties*** mean. Maybe mention MCU_GPIO_MODE_xxxx  and MCU_GPIO_FUNC_xxxx constants.

Reply:
Done.